### DISCUSSION TITLE
第4节课笔记(5班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#324] [笔记]
宝子们，五班的第四次笔记交这里哦

**Comment by shengmouget** [comment]
[XTuner 大模型单卡低成本微调实战](https://blog.csdn.net/shengweiit/article/details/135543963)

**Comment by DreamBinary** [comment]
https://github.com/DreamBinary/InternLMCamp/tree/master/day04/note

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135561645?spm=1001.2014.3001.5501

**Comment by GitEasonXu** [comment]
https://github.com/GitEasonXu/InternLM/blob/main/homework/course_4/coursenote.md

**Comment by JayceNing** [comment]
https://zhuanlan.zhihu.com/p/677322022

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class4-note/class4_note.md

**Comment by Milkve** [comment]
[第四次笔记](https://juejin.cn/spost/7324253082615889971)

**Comment by angellyao** [comment]
第四次笔记https://zhuanlan.zhihu.com/p/676826321?

**Comment by Farewell-CK** [comment]
第四次笔记https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE/note.md
